sockeye/train.py

```diff
@@ -497,20 +554,46 @@ def get_num_embed(args: argparse.Namespace) -> Tuple[int, int]:
                             "has to be even and a multiple of encoder attention heads (%d)" % (
                                 adjusted_transformer_encoder_model_size, args.transformer_attention_heads[0]))
 
+    if not num_embed_source:
+        num_embed_source = C.DEFAULT_NUM_EMBED
+
     if args.decoder == C.TRANSFORMER_TYPE:
```

Comment:
mhhh btw how does this work with SSRU decoders?
This check seems to be a left over from Sockeye 1 where we branched on Transformers vs RNNs, right?
Would it make sense to remove it?

Reply:
good catch. I'll remove the branch, also for encoders

Reply:
thanks!